diff --git a/docroot/sites/all/modules/contrib/miniorange_saml_idp/includes/metadata/metadata.php b/docroot/sites/all/modules/contrib/miniorange_saml_idp/includes/metadata/metadata.php
index 00717e0..1f693f3 100755
--- a/docroot/sites/all/modules/contrib/miniorange_saml_idp/includes/metadata/metadata.php
+++ b/docroot/sites/all/modules/contrib/miniorange_saml_idp/includes/metadata/metadata.php
@@ -1,11 +1,11 @@
 <?php
-define('DRUPAL_ROOT', dirname(dirname(dirname(dirname(dirname(dirname(dirname(__FILE__))))))));
+define('DRUPAL_ROOT', dirname(dirname(dirname(dirname(dirname(dirname(dirname(dirname(__FILE__)))))))));
 
 require_once DRUPAL_ROOT . DIRECTORY_SEPARATOR . 'includes' . DIRECTORY_SEPARATOR . 'bootstrap.inc';
 drupal_bootstrap(DRUPAL_BOOTSTRAP_FULL);
-  
-global $base_url;	
-$site_url = substr($base_url, 0, strpos($base_url, 'sites'));   
+
+global $base_url;
+$site_url = substr($base_url, 0, strpos($base_url, 'sites'));
 
 $entity_id = $site_url . '?q=admin/config/people/miniorange_saml_idp/';
 $login_url = $site_url . '?q=initiatelogon';
@@ -19,9 +19,9 @@ $certificate = str_replace( " ", "", $certificate );
 
 
 header('Content-Type: text/xml');
-echo'<?xml version="1.0" encoding="UTF-8"?><md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" entityID="'.$entity_id.'"><md:IDPSSODescriptor WantAuthnRequestsSigned="false" protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol"><md:KeyDescriptor 
+echo'<?xml version="1.0" encoding="UTF-8"?><md:EntityDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" entityID="'.$entity_id.'"><md:IDPSSODescriptor WantAuthnRequestsSigned="false" protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol"><md:KeyDescriptor
 			use="signing">
-			<ds:KeyInfo 
+			<ds:KeyInfo
 				xmlns:ds="http://www.w3.org/2000/09/xmldsig#">
 				<ds:X509Data>
 					<ds:X509Certificate>'.$certificate.'</ds:X509Certificate>
@@ -31,11 +31,11 @@ echo'<?xml version="1.0" encoding="UTF-8"?><md:EntityDescriptor xmlns:md="urn:oa
 
 		<md:NameIDFormat>urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress</md:NameIDFormat>
 		<md:NameIDFormat>urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified</md:NameIDFormat>
-		<md:SingleSignOnService 
-			Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" 
+		<md:SingleSignOnService
+			Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST"
 			Location="'.$login_url.'"/>
-		<md:SingleSignOnService 
-			Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" 
+		<md:SingleSignOnService
+			Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect"
 			Location="'.$login_url.'"/>
 	</md:IDPSSODescriptor>
-</md:EntityDescriptor>';
\ No newline at end of file
+</md:EntityDescriptor>';
diff --git a/docroot/sites/all/modules/contrib/miniorange_saml_idp/miniorange_sp_information.inc b/docroot/sites/all/modules/contrib/miniorange_saml_idp/miniorange_sp_information.inc
index bc7d5bc..e804afc 100755
--- a/docroot/sites/all/modules/contrib/miniorange_saml_idp/miniorange_sp_information.inc
+++ b/docroot/sites/all/modules/contrib/miniorange_saml_idp/miniorange_sp_information.inc
@@ -13,10 +13,11 @@
   $issuer = $base_url . '/?q=admin/config/people/miniorange_saml_idp/';
   $url = $base_url . '/';
   $login_url = $url . '?q=initiatelogon';
-  
+  $module_path = drupal_get_path('module', 'miniorange_saml_idp');
+
   $mo_admin_email = variable_get('miniorange_saml_idp_customer_admin_email',NULL);
   $sml_lk = variable_get('miniorange_saml_idp_sml_lk', NULL);
-  
+
   if(!isset($mo_admin_email)){
     $form['markup_saml_idp_regsitration_message'] = array(
 	  '#markup' => '<div style="display:block;margin-top:10px;color:red;background-color:rgba(251, 232, 0, 0.15);'
@@ -32,7 +33,7 @@
 	  . ' to enable Drupal IDP.</div>'
 	);
   }
-  
+
   $form['header'] = array(
     '#markup' => '<center><h3>You will need the following information to'
     . ' configure your Service Provider. Copy it and keep it handy</h3></center>',
@@ -62,7 +63,7 @@
 
   $options[3] = array(
     'attribute' => t('Certificate (Optional)'),
-    'value' => t('<a href="' . $base_url . '/sites/all/modules/miniorange_saml_idp/resources/idp-signing.crt">Download</a>'),
+    'value' => t('<a href="' . $base_url . '/' . $module_path . '/resources/idp-signing.crt">Download</a>'),
   );
 
   $options[4] = array(
@@ -93,7 +94,7 @@
 
   $form['markup_idp_sp_2'] = array(
     '#markup' => '<code style="background-color:gainsboro;"><b>'
-	. '<a target="_blank" href="' . $base_url . '/sites/all/modules/miniorange_saml_idp/includes/metadata/metadata.php">' . $base_url . '/sites/all/modules/miniorange_saml_idp/includes/metadata/metadata.php' . '</a></b></code>',
+	. '<a target="_blank" href="' . $base_url . '/' . $module_path . '/includes/metadata/metadata.php">' . $base_url . '/' . $module_path . '/includes/metadata/metadata.php' . '</a></b></code>',
   );
 
   return $form;
